hpotter/plugins/ListenThread.py

```diff
@@ -92,6 +92,8 @@ def run(self):
             source = None
             try:
                 source, address = listen_socket.accept()
+                timeout = self.config.get('accepted_socket_timeout')
+                source.settimeout(timeout)
```

Comment:
set timeout on accepted connection listening to attacker. if the key is not in config, defaults to None